src/Model/Anime/Anime.php

```diff
@@ -57,6 +57,11 @@ class Anime
      */
     private $titleSynonyms;
 
+    /**
+     * @var \Jikan\Model\Common\Title[]
+     */
+    private $titles;
```

Comment:
We are targeting php 8.0 afaik, so you can write `private array $titles;` I think.
If we indeed target php 8.0 then same applies for other added member variables below.